snippets/tomorrow.md

```diff
@@ -1,10 +1,20 @@
 ### tomorrow
 
 Results in a string representation of tomorrow's date.
-Use `new Date()` to get today's date, adding `86400000` of seconds to it(24 hours), using `Date.toISOString()` to convert Date object to string.
+Use `new Date()` to get today's date, adding one day using `Date.getDate()` and `Date.setDate()`, and converting the Date object to a string.
 
 ```js
-const tomorrow = () => new Date(new Date().getTime() + 86400000).toISOString().split('T')[0];
+const tomorrow = () => {
+  let t = new Date();
+  t.setDate(t.getDate() + 1);
```

Comment:
Hey @NielsLeenheer 👋 😄 

Thanks for taking the time to submit this PR

However here are my suggestions 

Rather than spreading everything out and making `-` appear on their own line.

Why not keep things nice & short as possible by doing some variant of the following. 

```js
const tomorrow = () => {
  let time = new Date();
  let month = `${time.getMonth()+1}`.padStart(2, '0');
  let day = `${time.getDate()+1}`.padStart(2, '0');
  return `${time.getFullYear()}-${month}-${day}`;
};
```

Reply:
This will definitely break on the last day of each month, as it will go from `Dec 31` to `Dec 01`. Btw in this example, the year will also be broken. A way to make it shorter would be:

```js
const tomorrow = () => {
  let t = new Date();
  t.setDate(t.getDate() +1);
  return `${t.getFullYear()}-${String(t.getMonth() + 1).padStart(2, '0')}-${String(t.getDate()).padStart(2, '0')}`;
};
```

Reply:
@Chalarangelo ahhh yea, Nice catch!

I quickly came up with a sloppy solution. I am sure this can be refactored. But it takes care of the edge cases you mentioned
```js
const tomorrow = () => {
  let time = new Date();
  let maxDay = { 01: 31, 02: 28, 03: 31, 04:30, 05: 31, 06: 30, 07: 31, 08:31, 09: 30, 10: 31, 11: 30, 12: 31}; 
  let month = `${time.getMonth()+1}`.padStart(2, '0');
  let nextMonth = `${time.getMonth()+2}`.padStart(2, '0');
  let day = `${time.getDate()}`.padStart(2, '0');
  let isLastDayOfMonth = maxDay[month] === day;
  let tomorrow = `${time.getDate()+1}`.padStart(2, '0');
  let isNewYearEve = month == 12 && maxDay[month] === day;
  let year = isNewYearEve ? time.getFullYear() + 1 : time.getFullYear();
  return `${year}-${isNewYearEve ? 01 : isLastDayOfMonth ? nextMonth : month}-${isLastDayOfMonth ? 01 : tomorrow}`;
};
```

Reply:
@kingdavidmartins and @Chalarangelo 

The use of 
```js
t.setDate(t.getDate() + 1);
```
on a separate line is very deliberate. 

This will **not** break on month or year boundaries. Using this line on `Nov 30` will in fact set it to `Dec 1` and not `Nov 1`. It will also take care of year boundaries automatically and update `Dec 31, 2017` to `Jan 1, 2018`. This behaviour is in the spec. This is exactly what the `Date` object should do and every implementation follows this.

So you should not handle the boundaries yourself, they `Date` object does that for you. But if you do, you are bound to make mistakes due to wrong assumptions. For example in that last solution there is an assumeption that every February will have 28 days. But it doesn't due to leap days. 

@Chalarangelo 

Using templates to generate the return value is indeed shorter and changing that would be okay. But in my opinion it does hurt readability. And the non-templated version is just 4 bytes longer.

Reply:
:+1: on the Date rollover functionality, that is very often forgotten about. Should not be changed.

As for "shortness" I think moving to the template will help enough. When that change happens it should be fine

Reply:
example of rollover taken from console of Chrome:
```js
const b = new Date()
b //Sun Dec 31 2028 22:42:13 GMT-0800 (Pacific Standard Time)
b.setDate(32) //1862030533926
b //Mon Jan 01 2029 22:42:13 GMT-0800 (Pacific Standard Time)
```

Reply:
@NielsLeenheer Woah! totally forgot. Nice!! Thanks for pointing that out 🎉 ⚡️ 

Haven't really been reviewing dates documentation or using native dates methods as much since momentjs 😞 

Def going to go over the documentation more in depth later

Can confirm I also get the rollover result.

As for using template literals, I'm with @skatcat31.  Even though it will not be significatly shorter. It still will be shorter, have less lines, and promote the use of ECMAScript 2015

Update using template literals and should be good to go 😄 

Suggestion 1 ~ More readable

```js
const tomorrow = () => {
  let t = new Date();
  t.setDate(t.getDate() + 1);
  let month = `${t.getMonth()+1}`.padStart(2, '0');
  let day = `${t.getDate()}`.padStart(2, '0');
  return `${t.getFullYear()}-${month}-${day}`;
};
```

Suggestion 2 ~ Less lines

```js
  let t = new Date();
  t.setDate(t.getDate() + 1);
  return `${t.getFullYear()}-${String(t.getMonth()+1).padStart(2, '0')}-${String(t.getDate()).padStart(2, '0')}`;
};
```